scripts/actions/fetch-observability-packs.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+/**
+ * This script is used to query the New Relic GraphQL API for Observability Packs.
+ * It then writes the array of Observability Packs to src/data/observability-packs.json
+ * It requires the following environment variables to be set:
+ * NR_GQL_URL - The New Relic GraphQL URL
+ * NR_API_TOKEN - A New Relic personal API token
+ **/
+
+/* eslint-disable no-console */
+const fs = require('fs');
+const fetch = require('node-fetch');
+
+const PACKS_FILE_PATH = './src/data/observability-packs.json';
+const NR_GQL_URL = process.env.NR_GQL_URL;
+const NR_API_TOKEN = process.env.NR_API_TOKEN;
+
+const packQuery = `# gql 
+  {
+    docs {
+      openInstallation {
+        observabilityPackSearch {
+          count
+          results {
+            observabilityPacks {
+              authors
+              dashboards {
+                description
+                name
+                screenshots
+                url
+              }
+              description
+              iconUrl
+              id
+              level
+              logoUrl
+              name
+              websiteUrl
+            }
+          }
+        }
+      }
+    }
+  }
+`;
+
+/**
+ * Queries graphql for the provided query
+ * @param {String} queryString the graphql query to send
+ * @param {String} url NR graphql endpoint
+ * @param {String} token NR api token
+ * @returns {Promise<Object[]|undefined>} returns the resulting array
+ * or `undefined` if there was an error
+ **/
+const fetchPacks = async (queryString, url, token) => {
+  try {
+    const results = await fetch(url, {
+      method: 'post',
+      body: JSON.stringify({ query: queryString }),
+      headers: {
+        'Content-Type': 'application/json',
+        'Api-Key': token,
+      },
+    }).then((res) => res.json());
+
+    if (results.errors) {
+      console.log('GRAPHQL Errors:', JSON.stringify(results.errors, null, 2));
+    }
+
+    return results.data.docs.openInstallation.observabilityPackSearch.results;
```

Comment:
Just to be defensive, we might want to use something like [`_.get`](https://lodash.com/docs/4.17.15#get) (if we have lodash available) to ensure we don't hit an error here. Maybe the `catch` is sufficient tho, I dunno.

Reply:
I was planning on letting the `try/catch` handle that case, it will catch a `cannot access property of undefined` error. I also don't mind being more explicit here

Reply:
@zstix Do you think it's worth pulling in lodash?

Reply:
We probably don't need _all_ of lodash. If we just pull in `get` itself, then we're only looking at adding [2 - 4kb](https://bundlephobia.com/package/lodash.get@4.4.2) to our bundle.

Reply:
https://www.npmjs.com/package/lodash.get

Reply:
Done!